pkgs/applications/science/chemistry/d-seams/default.nix

```diff
@@ -0,0 +1,32 @@
+{ clangStdenv, stdenv, fetchurl, catch2, rang, fmt, libyamlcpp, cmake, eigen
+, lua, luaPackages, liblapack, blas, lib, boost, gsl ? null }:
```

Comment:
gsl is not a big library. Would it make sense to just always build with gsl support?

Reply:
Yup, excellent point, done ^_^